src/main/java/seedu/address/model/person/Family.java

```diff
@@ -0,0 +1,73 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Represents the family size of a person in the address book.
+ * Guarantees: family size is present and not null, and adheres to specific constraints.
+ */
+public class Family {
+
+    /** Message for constraints on family size. */
+    public static final String MESSAGE_CONSTRAINTS = "Family size should be at least one";
+
+    /** The family size. */
+    private Integer familySize;
+
+    /**
+     * Constructs a {@code Family} instance with the given family size.
+     *
+     * @param familySize The size of the family.
```

Comment:
just thinking: maybe we should include that the size of the family includes oneself in the comments?

Reply:
Changed and I will add assertion in `parseFamily(String family)` of ParserUtil